src/include/network/lan/cmdline.rb

```diff
@@ -125,58 +126,37 @@ def getItem(options, config)
     # Handler for action "show"
     # @param [Hash{String => String}] options action options
     def ShowHandler(options)
-      config = getConfigList("")
-      return false unless validateId(options, config)
-
-      required_id = options["id"]
-      config.each do |row|
-        row.each_pair do |key, value|
-          next if key != required_id
-
-          # create plain text from formated HTML
-          descr = value["rich_descr"]
-          descr.gsub!(/(<br>)|(<\/li>)/, "\n")
-          descr.gsub!(/<[^>]+>/, "")
+      config = Yast::Lan.yast_config
+      return false unless validateId(options, config.interfaces)
 
-          # to keep it backward compatible. Previously echo was called which adds additional new line"
-          descr << "\n"
+      presenter = Y2Network::Presenters::InterfaceSummary.new(config.interfaces.to_a[options["id"].to_i].name, config)
+      text = presenter.text
+      # create plain text from formated HTML
```

Comment:
NP: maybe something for the presenter? like plain_text method

Reply:
well, also list handler deserve own presenter, but due to its current state, I prefer easy and straighforward fix. Also looks like CLI is not much used, so goal is clear. Make it working, but do not invest much.